src/components/Good.tsx

```diff
@@ -0,0 +1,7 @@
+type GoodProps = {
+  goodName: string;
+};
+
+export const Good: React.FC<GoodProps> = ({ goodName }) => {
+  return <li data-cy="Good">{goodName}</li>;
+};
```

Comment:
it's redundant to create so simple component, better to write it directly inside GoodsList